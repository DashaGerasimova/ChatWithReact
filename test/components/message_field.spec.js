import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import MessageField from '../../app/containers/chat/components/message_field/index'

describe('MessageField component', () => {
  it('should render field', () => {
    const tree = toJson(shallow(<MessageField submit={()=>{}} />));
    expect(tree).toMatchSnapshot();
  });

  const test_submit = (text) => {
  	
  }  
	
  // it('should ', () => {
  //   const field = shallow(<MessageField submit={test_submit} />)
  //   const tree = renderer.create(
  //     <List messages={items} />
  //   ) .toJSON();
  // expect(tree).toMatchSnapshot();
  // });
})