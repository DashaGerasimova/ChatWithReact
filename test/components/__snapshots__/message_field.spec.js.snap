// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageField component should render field 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input />
    <button>
      Add Message
    </button>
  </form>
</div>
`;
